gui/object.cpp

```diff
@@ -59,6 +59,12 @@ void GuiObject::resize(int x, int y, int w, int h, bool scale) {
 	}
 }
 
+Widget *GuiObject::addChild(Widget *newChild) {
```

Comment:
maybe we should call setNext here instead of trusting the caller (Widget::init) to handle that?